SimCalorimetry/HGCalSimProducers/plugins/HGCDigitizer.cc

```diff
@@ -384,7 +384,7 @@ void HGCDigitizer::accumulate_forPreMix(PileUpEventPrincipal const& e,
                                         edm::EventSetup const& eventSetup,
                                         CLHEP::HepRandomEngine* hre) {
   edm::Handle<edm::PCaloHitContainer> hits;
-  e.getByLabel(edm::InputTag("g4SimHits", hitCollection_), hits);
+  e.getByLabel(hitTag_, hits);
```

Comment:
Why `getByLabel`?

Reply:
PileUpEventPrincipal allows on getByLabel. I asked Matti and he said token access is not there right now. It may happen later.